tests/fault_tolerance/deploy/test_deployment.py

```
@@ -0,0 +1,172 @@
+    parse_results(
+        logs_dir=None,
+        log_paths=global_result_list,
+        tablefmt="fancy_grid",
+    )
+
+
+@pytest.mark.e2e
+@pytest.mark.slow
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
+async def test_fault_scenario(
+    scenario,  # noqa: F811
```

Ruff (0.13.1)

Unused `noqa` directive (unused: `F811`)

Remove unused `noqa` directive

(RUF100)